Naughts_and_Crosses.py

```diff
@@ -0,0 +1,114 @@
+board_state = [['| |','| |','| |'],['| |','| |','| |'],['| |','| |','| |']]
+win_condition = ()
+def printboard(list):
+	for l in list:
+		print(''.join(l))
+
+def end_game():
+	if board_state == win_condition:
+		print('Well Done, you won! Good Job')
+	else:
+		print('Oh dear, looks like a draw...')
+	print('Would you like to play again?')
+
+
+
+class Player:
+	def __init__(self,name):
+		Player.name = name
+	def choice_row():
+		x=()
+		while x not in range(0,3):
+			x= input('Pick what row you would like') - 1
+			if x not in range(0,3):
+				print('That\'s not in the right format you douche, try again')
+		return x
+
+	def choice_column():
+		y=()
+		while y not in range(0,3):
+			y= input('Pick what column you would like') - 1
+			if y not in range(0,3):
+				print('That\'s not in the right format you douche, try again')
+				y = ()
+		return y
+	def turn():
+		turn = 'On-Going'
+		while turn == 'On-Going':
+			x_row = choice_row()
+			y_column = choice_column()
+			if board_state[x_row][y_column] == '|X|' or board_state[x_row][y_column] == '|O|':
```

Comment:
```python
if board_state[x_row][y_column] == ('|X|' or '|O|'):
    ...
# Or better yet
if board_state[x_row][y_column] in ['|X|', '|O|']:
    ...
```

Really you need to be only checking for X or O, its mad including the '|' chars. Then you could check to see if the square is blank, rather than this.